plan/match_property.go

```diff
@@ -252,6 +252,13 @@ func (p *Projection) matchProperty(_ *requiredProperty, childPlanInfo ...*physic
 }
 
 // matchProperty implements PhysicalPlan matchProperty interface.
+func (p *TempStore) matchProperty(prop *requiredProperty, childPlanInfo ...*physicalPlanInfo) *physicalPlanInfo {
+	np := *p
```

Comment:
you don't need to implent this function. Just "panic("You can't call this function!")"